Sources/QuaternionModule/README.md

```diff
@@ -4,7 +4,7 @@ This module provides a `Quaternion` type generic over an underlying `RealType`:
 
 ```swift
 1> import QuaternionModule
-2> let q = Quaternion(1, (1,1,1)) // q = 1 + i + j + k
+2> let q = Quaternion(real: 1, imaginary: SIMD3(1, 1, 1)) // q = 1 + i + j + k
```

Comment:
The README should probably highlight the default non-simd initializer.

```suggestion
2> let q = Quaternion(real: 1, imaginary: 1, 1, 1) // q = 1 + i + j + k
```

Reply:
Thank you! New to swift and forgot to try the simplest thing! :D